### DISCUSSION TITLE
no mention of target:"ie11" anywhere in the docs

### DISCUSSION THREAD

**Discussion by MathGeniusJodie** [tailwindlabs/tailwindcss#2154] [General]
maybe this should be an issue on the tailwind project itself?
I just found out about this feature from reading the docs to a plugin, is there a reason why it isn't officially documented yet? is this feature stable?

**Comment by estevanmaito** [comment]
There's a related open issue here https://github.com/tailwindlabs/tailwindcss.com/issues/431 (used to be `docs` repo)

You can read more about it here 👇:

Release notes: https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.4.0#ie-11-target-mode

Original PR: https://github.com/tailwindlabs/tailwindcss/pull/1635

---

I'll take the opportunity to point this out (part of the release notes):

> This feature is experimental and may change outside of normal semantic versioning policies.

Couldn't this option be moved to the new `experimental` config property @adamwathan ?

**Reply by adamwathan** [reply]
It could yeah, if we think it's stable enough though that would be a breaking change for no reason, so going to think about it a bit before doing that.

**Reply by MathGeniusJodie** [reply]
aight, I'll wait before using it in my plugin
